torch/csrc/autograd/profiler_legacy.cpp

```diff
@@ -226,6 +226,7 @@ void ProfilerThreadLocalState::pushRange(
     evt.setSequenceNr(fn.seqNr());
     evt.setFwdThreadId(fn.forwardThreadId());
     evt.setScope((uint8_t)fn.scope());
+    evt.setExtra(fn);
```

Comment:
let's here call utility function to extra extra args from RecordFunction and then call  setter method on LegacyEvent passing in the map

Reply:
and after this call setFlops(computeFlops(map))